python/ola/ClientWrapper.py

```diff
@@ -44,8 +44,30 @@ def __init__(self, time_ms, callback):
                     datetime.timedelta(milliseconds=time_ms))
     self._callback = callback
 
-  def __cmp__(self, other):
-    return cmp(self._run_at, other._run_at)
+  def __eq__(self, other):
+    if other.__class__ is not self.__class__:
+      return False
+    return self._run_at == other._run_at and self._callback == other._callback
+
+  def __lt__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
+    return self._run_at < other._run_at
+
+  def __le__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
+    return self._run_at <= other._run_at
```

Comment:
Can we change this to do the Python style, so < or `__eq__`:
https://github.com/python/cpython/blob/0baa72f4b2e7185298d09cf64c7b591efcd22af0/Lib/asyncio/events.py#L97-L145